avm/ptn/deployment-script/import-image-to-acr/scripts/importscript.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+echo 'Waiting on RBAC replication ($initialDelay)'
+sleep $initialDelay
+
+# retry loop to catch errors (usually RBAC delays, but 'Error copying blobs' is also not unheard of)
+retryLoopCount=0
+until [ $retryLoopCount -ge $retryMax ]
```

Comment:
`retyMax` is never intiated, right?

Reply:
It is passed into the environment and is available.